src/osp/Satellites/SatActiveArea.h

```diff
@@ -32,6 +32,9 @@ struct UCompActiveArea
     //active::ActiveEnt m_camera;
 
     unsigned m_sceneIndex;
+
+    // true when the ActiveArea is moving
+    bool m_inMotion;
```

Comment:
Does "ActiveArea is moving" mean a discrete movement or that the frame of reference is actually moving with respect to the system frame?

Reply:
This bool doesn't do anything yet. It is suppose to mean that the entire scene frame of reference is moving through the universe. This will likely be removed in favour of a more descriptive enum enum of possible states an active area can be in.

The two possible states planned is:
* Locked to surface: frame of reference is locked to the surface of a planet, and will rotate with it.
* In Deep space: Frame of reference can have a velocity. Orbital velocity can be like 9000m/s; without a velocity, a floating origin translation has to be done every frame.